docs/packaging/your-first-package.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Your First Package
+summary: Your First Package
+---
+
+# Your First Package
+
+This page will guide you through the steps required to build your first package. This is really a test to make sure that your [build environment is set up correctly](docs/packaging/index.md). You will clone an existing package, build it, then install it, and finally remove it.
+
+## Cloning and building `nano`
+
+For this example we will use the text editor `nano`.
+
+Move to your build directory. Your directory may have a different name or location:
+
+```bash
+cd ~/solus-builds
+```
+
+Clone the `nano` repository, then move into it:
+
+```bash
+make nano.clone
```

Comment:
I don't think it is immediately obvious what this will do. Does this clone from the repo?

Reply:
Yes, this clones from github. Are you suggesting that this should be explained?

Reply:
Yes, I think it should explain that this will clone from our GitHub solus-packages repository

Reply:
It might be worth having this not using `make` but rather gh cli calls directly to start helping them familiarize themselves with it. The go task stuff is nice for those that are doing this on the daily but I'm concerned it overcomplicates / obfuscates processes in onboarding.

I agree that emphasis that this is upstream rather than a fork is worth getting into, but not sure if it should be done here :thinking: